php/class-customize-snapshot-manager.php

```diff
@@ -640,6 +645,62 @@ public function render_templates() {
 
 			<# } #>
 		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict-button">
+			<?php $title_text = __( 'has potential conflicts (click to expand)', 'customize-snapshots' ); ?>
```

Comment:
This will be problematic to translate.

Reply:
What if a language is Object-Verb-Subject order? The placeholder should be inside the translation string.